app/controllers/v0/sessions_controller.rb

```diff
@@ -33,6 +33,9 @@ def saml_callback
       if @saml_response.is_valid? && persist_session_and_user
         async_create_evss_account(@current_user)
         redirect_to SAML_CONFIG['relay'] + '?token=' + @session.token
+
+        obscure_token = Session.obscure_token(@session.token)
+        Rails.logger.info("Logged in user with id #{@session.uuid}, token #{obscure_token}")
```

Comment:
That helps me understand.  I'll try to re-iterate to solidify my understanding:

```ruby
# lets us see the flow for a single HTTP request
[1e7c4ad6-0512-468e-b46d-f6fd40733e0b] INFO : blah blah blah
```
```ruby
# lets us tie all requests made by a user in a single login session
[1e7c4ad6-0512-468e-b46d-f6fd40733e0b] [ab530a13e4ab530a13e4] INFO : blah blah blah
[aaaaaaaa-5555-7777-eeee-ffffffffffff] [ab530a13e4ab530a13e4] INFO : blah blah blah
```
```ruby
# lets us tie all requests made by a user across all of their sessions
[1e7c4ad6-0512-468e-b46d-f6fd40733e0b] [aaaaaaaaaaaaaaaaaaaa] INFO : Logged in user with id dead-beef-dead-beef, token aaaaaaaaaaaaaaaaaaaa
[13456789-4567-cccc-dddd-ffffffffffff] [aaaaaaaaaaaaaaaaaaaa] INFO : blah blah blah
[eeeeeeee-0111-85fd-feee-1234567890fd] [bbbbbbbbbbbbbbbbbbbb] INFO : Logged in user with id dead-beef-dead-beef, token bbbbbbbbbbbbbbbbbbbb
[bbbbbbbb-2222-451f-dcba-7654321fdca0] [bbbbbbbbbbbbbbbbbbbb] INFO : blah blah blah
```
Tracking within a single session makes sense, but across all of their sessions seems like a lot.

Also, the line:
```ruby
Rails.logger.info("Logged in user with id #{@session.uuid}, token #{obscure_token}")
```
the `obscure_token` is redundant because it is already added by `config.log_tags`